java8_time/year.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package java8_time
+
+type Year struct {
+	Year int32 `hessian:"year"`
+}
+
+func (Year) JavaClassName() string {
+	return "com.alibaba.com.caucho.hessian.io.java8.YearHandle"
```

Comment:
1.  is it only support the https://github.com/apache/dubbo-hessian-lite?    
2. is there some dubbo users use some other hessian libraries?  Yes, as I know.
3. should we support class name `java.time.Year` at the same time?

Reply:
Hi @wongoo, sorry, we can not fully understand these questions.

> 1. is it only support the https://github.com/apache/dubbo-hessian-lite?

Yes, we implemented these code by following the dubbo-hessian-lite.

> 2. is there some dubbo users use some other hessian libraries? Yes, as I know.

Could you please offer the libraries names?

> 3. should we support class name java.time.Year at the same time?

When shall we use `java.time.Year` instead of `com.alibaba.com.caucho.hessian.io.java8.YearHandle`?

Reply:
some companies use the original hessian java library,  if u insist it's only for dubbo-hessian-lite, then no problems.